python/cugraph/cugraph/tests/mg/test_mg_property_graph.py

```diff
@@ -866,11 +866,14 @@ def test_renumber_vertices_by_type(dataset1_MGPropertyGraph, prev_id_column):
         assert df_id_ranges.loc[key, "stop"] == stop
         df = pG.get_vertex_data(types=[key]).compute()
         assert len(df) == stop - start + 1
-        assert (df["_VERTEX_"] == list(range(start, stop + 1))).all()
+        assert (
+            df["_VERTEX_"] == df["_VERTEX_"]._constructor(range(start, stop + 1))
+        ).all()
```

Comment:
These changes are similar to https://github.com/rapidsai/cugraph/pull/3036/files#r1048542911 .

 Unsure how CI passed without them